### PR TITLE
DTFS2-2071 BSS - table overflow

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Changes made: 

* For BSS tables, added a div with overflow class
* Added CSS style for overflow-x

### CODE REVIEW

**Review by IgnatG — DISMISSED** [review]

**Review by ttbarnes — DISMISSED** [review]

File: portal/templates/contract/components/contract-overview-table.njk [hunk 1]
```diff
@@ -4,68 +4,71 @@
...
 
-    <tbody class="govuk-table__body">
-      <tr class="govuk-table__row">
-        <td data-cy="bankInternalRefName" class="govuk-table__cell govuk-!-font-size-14">
-          {{ deal.bankInternalRefName | dashIfEmpty }}
-        </td>
+      <tbody class="govuk-table__body">
+        <tr class="govuk-table__row">
+          <td data-cy="bankInternalRefName" class="govuk-table__cell govuk-!-font-size-14">
+            {{ deal.bankInternalRefName | dashIfEmpty }}
```
Comment (abhi-markan) [inline portal/templates/contract/components/contract-overview-table.njk:31]: Please add `| safe` filter along all displayed items.

File: portal/templates/contract/components/loan-transactions-table.njk [hunk 2]
```diff
@@ -11,80 +11,81 @@
...
+              <tr class="govuk-table__row" data-cy="loan-{{ loan._id }}">
+                <td class="govuk-table__cell govuk-!-font-size-14">
+                  {{ loanLink.render(params, loan) }}
+                </td>
 
-              <td class="govuk-table__cell govuk-!-font-size-14" data-cy="loan-ukef-facility-id-{{ loan._id }}">
-                {{ loan.ukefFacilityId }}
-              </td>
+                <td class="govuk-table__cell govuk-!-font-size-14" data-cy="loan-ukef-facility-id-{{ loan._id }}">
+                  {{ loan.ukefFacilityId }}
```
Comment (abhi-markan) [inline portal/templates/contract/components/loan-transactions-table.njk:38]: `safe |` filter for all display cells.

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: portal/templates/contract/components/loan-transactions-table.njk (hunk 2 above)
Comment (Zainzzkk) [reply portal/templates/contract/components/loan-transactions-table.njk]: done

File: portal/templates/contract/components/contract-overview-table.njk (hunk 1 above)
Comment (Zainzzkk) [reply portal/templates/contract/components/contract-overview-table.njk:31]: done

Comment (abhi-markan) [inline portal/templates/contract/components/contract-overview-table.njk:31]: Missing filter here.

File: portal/templates/contract/components/contract-overview-table.njk [hunk 3]
```diff
@@ -4,68 +4,71 @@
...
+          </td>
 
-        <td data-cy="ukefDealId" class="govuk-table__cell govuk-!-font-size-14">
-          {{ details.ukefDealId | dashIfEmpty}}
-        </td>
+          <td data-cy="ukefDealId" class="govuk-table__cell govuk-!-font-size-14">
+            {{ details.ukefDealId | dashIfEmpty}}
```
Comment (abhi-markan) [inline portal/templates/contract/components/contract-overview-table.njk:35]: Missing filters on all the cells here.

**Review by abhi-markan — CHANGES_REQUESTED** [review]

File: portal/templates/contract/components/contract-overview-table.njk (hunk 1 above)
Comment (Zainzzkk) [reply portal/templates/contract/components/contract-overview-table.njk:31]: done

File: portal/templates/contract/components/contract-overview-table.njk (hunk 3 above)
Comment (Zainzzkk) [reply portal/templates/contract/components/contract-overview-table.njk:35]: done

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**